lightningd/lightningd.c

```diff
@@ -256,8 +311,11 @@ int main(int argc, char *argv[])
 
 	io_poll_override(debug_poll);
 
-	/* Figure out where we are first. */
-	ld->daemon_dir = find_my_path(ld, argv[0]);
+	/* Figure out where our daemons are first. */
+	ld->daemon_dir = find_daemon_dir(ld, argv[0]);
+	if (!ld->daemon_dir) {
+		errx(1, "Could not find daemons");
+	}
```

Comment:
I don't usually use {} around simple statements like this.

Reply:
I get very paranoid about dangling else problems and always use `{` `}` with `if`.